fuzzers/nyx_libxml2_parallel/src/main.rs

```diff
@@ -0,0 +1,103 @@
+use std::path::{Path, PathBuf};
+
+use libafl::bolts::launcher::Launcher;
```

Comment:
This could be one big `libafl::{` block

Reply:
Are there some tools that can merge the namespace of `use`?
Assume I have
```
use libafl::bolts::launcher::Launcher;
use libafl::bolts::rands::StdRand;
use libafl::bolts::shmem::ShMemProvider;
```
and I'd like to get the following result after running the tool:
```
use libafl::bolts::{
    launcher::Launcher,
    rands::StdRand,
    shmem::ShMemProvider,
};
```

Reply:
I found the tool to automate it. Add `imports_granularity = "Crate"` in `~/.config/rustfmt/rustfmt.toml` then `cargo fmt` will do everything for you.

Reply:
There is (was?) also `merge_imports = true` for nightly rust fmt, good to know. Can we add a rustfmt.toml to the repo?

Reply:
> There is (was?) also merge_imports = true for nightly rust fmt,

`merge_imports = true` is replaced by `imports_granularity = "Crate"`, see [this](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md), `cargo fmt` uses `rustfmt`, so `rustfmt.toml` will also work.
> Can we add a rustfmt.toml to the repo?

That's OK, just put it on the root of the project, we can do that in another PR.

Reply:
Nice👍